app-admin/fluent-bit/files/initd-1

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+name="fluent-bit daemon"
+description=""
+command=/usr/bin/fluent-bit
+command_args="-c ${fluent_conf:-/etc/fluent-bit/fluent-bit.conf} --daemon"
```

Comment:
```suggestion
command_args="-c ${FLUENT_CONF:-/etc/fluent-bit/fluent-bit.conf} --daemon"
```
And I find it easier to ship a **conf.d** file along with it, for easier modification.